dockerfiles/ubuntu/ubuntu-16.04/ubuntu-16.04-base/Dockerfile

```diff
@@ -16,7 +16,12 @@
 
 FROM ubuntu:16.04
 
-RUN apt-get update && \
+# apt-key is using fingerprint from https://packagecloud.io/github/git-lfs/gpgkey
+# wget -O - https://packagecloud.io/github/git-lfs/gpgkey | gpg --with-fingerprint
+RUN apt-get update && apt-get install -y apt-transport-https && \
+    echo "deb https://packagecloud.io/github/git-lfs/ubuntu/ xenial main" >> /etc/apt/sources.list && \
+    apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 6D398DBD30DD78941E2C4797FE2A5F8BDC282033 && \
```

Comment:
@rewitt1 `--recv-keys 6D398DBD30DD78941E2C4797FE2A5F8BDC282033` is the PGP fingerprint. It seemed better to hard code a [verifiable fingerprint](https://pgp.mit.edu/pks/lookup?search=packagecloud.io%2Fgithub%2Fgit-lfs&op=index&fingerprint=on) than to trust running a [shell script](https://packagecloud.io/github/git-lfs/install#bash-rpm) or carrying the gpg key as an artifact. This also applies for zypper because OpenSUSE has no packaging for git-lfs, at least not in the [15.1 release](https://software.opensuse.org/package/git-lfs). As soon as we feel comfortable dropping Ubuntu 16.04 and OpenSUSE 15.2 is released we can get rid of this packagecloud workaround.